examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/core_tests/test_airflow_asset_mapping.py

```diff
@@ -0,0 +1,56 @@
+from dagster._core.definitions.asset_key import AssetKey, CoercibleToAssetKey
+from dagster._core.definitions.asset_spec import AssetSpec
+from dagster._core.definitions.definitions_class import Definitions
+from dagster_airlift.constants import DAG_ID_METADATA_KEY, TASK_ID_METADATA_KEY
+from dagster_airlift.core.serialization.compute import build_task_spec_mapping_info
+
+
+def ak(key: str) -> AssetKey:
+    return AssetKey(key)
+
+
+def airlift_asset_spec(key: CoercibleToAssetKey, dag_id: str, task_id: str) -> AssetSpec:
```

Comment:
nit: can we move this to test utils for use across tests

Reply:
I have more things like to that add so will consolidate all in upstack.